### ISSUE TITLE
Make show information alert colors accessible

### ISSUE THREAD

**Issue by haietza** [gjb2048/moodle-format_topcoll#88]
Several of the text/background colors used on the assignment/quiz/choice/etc show information alerts are not accessible. Please consider modifying the styles to meet color contrast requirements.

If you are interested, these are the style overrides I put in place in our theme:

// Collapsed topics success alerts
.format-topcoll .ct-activity-date-submitted {
  background-color: #d4edda;
}

.format-topcoll .ct-activity-date-submitted a {
  color: #155724;
}

// Collapsed topics warning alerts
.format-topcoll .ct-activity-date-nearly-due {
  background-color: #FFF3CD;
}

.format-topcoll .ct-activity-date-nearly-due a {
  color: #856404;
}

// Collapsed topics danger alerts
.format-topcoll .ct-activity-date-overdue {
  background-color: #f8d7da;
}

.format-topcoll .ct-activity-date-overdue a {
  color: #721c24;
}

.format-topcoll .ct-activity-date-overdue .text-warning {
  color: #721C24 !important;
}

**Comment by gjb2048** [comment]
@haietza Thanks for this, I'll look as soon as I can.
